ignite/cmd/network_chain.go

```diff
@@ -9,7 +9,28 @@ import (
 func NewNetworkChain() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "chain",
-		Short: "Build networks",
+		Short: "Commands to launch chains",
+		Long: `The "chain" namespace features the most commonly used commands for launching
+blockchains with Ignite.
+
+As a coordinator you "publish" your blockchain to Ignite. When enough validators
+are approved for the genesis and no changes are excepted to be made to the
+genesis, a coordinator announces that the chain is ready for launch with the
+"launch" command. The chain launch can be cancelled with the "revert-launch"
+command.
```

Comment:
```suggestion
"launch" command. In the case of an unsuccessful launch, the coordinator can revert it using the "revert-launch"
command.
```

It doesn't cancel the chain launch, it allows to push the chain back to the coordination phase, after genesis time reached, in case the chain didn't launch